src/dsymbol.d

```diff
@@ -1207,6 +1207,9 @@ private:
     Dsymbols* importedScopes;
     PROTKIND* prots;            // array of PROTKIND, one for each import
 
+    import std.bitmanip : BitArray;
+    BitArray accessiblePackages;// whitelist of accessible (imported) packages
```

Comment:
@MartinNowak Indeed today the circular dependency between dmd and druntime is there, and it's difficult to maintain, but it's not so much problem because core language change is rare. If we'll add one more circular between phobos, it will make everything unmaintainable. Do not do it.